tests/TestCase/Database/Type/DateTimeTypeTest.php

```diff
@@ -146,9 +146,11 @@ public function testToDatabase()
         $result = $this->type->toDatabase($date, $this->driver);
         $this->assertEquals('2013-08-12 15:16:17', $result);
 
+        $tz = $date->getTimezone();
         $this->type->setTimezone('Asia/Kolkata'); // UTC+5:30
         $result = $this->type->toDatabase($date, $this->driver);
         $this->assertEquals('2013-08-12 20:46:17', $result);
+        $this->assertEquals($tz, $date->getTimezone());
```

Comment:
I used assertEquals() here, as DateTime::getTimezone() seems to create different TimeZone instances each time it is called.